README.md

```diff
@@ -111,45 +160,69 @@ curl -X POST \
      http://localhost:7777/rpc/v0
 ```
 
-## Minting an NFT
+### Minting an NFT
 
-In the example below, we've already created a deal for the 3 assets (metadata, thumbnail, and the original asset respectively) that comprise our NFT. These are as follows, with their corresponding CIDs.
+In the example below, we've already created a deal for the 3 assets (metadata, thumbnail, and the original asset respectively) that comprise our NFT. These are as follows, with their corresponding Content Identifiers (aka CIDs).
```

Comment:
Nice! Always a fan of defining acronyms and assuming no knowledge.  Technically, line 154 is the first usage, but either spot is probably fine.

Reply:
I'll move it to line 154!